dali/python/nvidia/dali/external_source.py

```diff
@@ -664,18 +664,33 @@ def external_source(source = None, num_outputs = None, *, cycle = None, name = N
     which is more performant.
     """
 
-    if batch is None:
-        batch = True
+    from nvidia.dali._debug_mode import _PipelineDebug
+
+    def _external_source(source = None, num_outputs = None, *, cycle = None, name = None, device = "cpu", layout = None,
+                    cuda_stream = None, use_copy_kernel = None, batch = True, **kwargs):
```

Comment:
Since you're touching it, please reformat that part too

Reply:
done